client/components/sites-popover/index.jsx

```diff
@@ -39,31 +37,11 @@ export default class extends React.Component {
 		className: '',
 	};
 
-	state = {
-		popoverVisible: false,
-	};
-
-	componentDidMount() {
-		this.updatePopoverVisibilityState();
-	}
-
-	componentDidUpdate( prevProps ) {
-		if ( this.props.visible !== prevProps.visible ) {
-			this.updatePopoverVisibilityState();
-		}
-	}
-
-	updatePopoverVisibilityState = () => {
-		this.setState( {
-			popoverVisible: this.props.visible,
```

Comment:
nice cleanup!
It definitely doesn't make sense for a component to hold `state` that is just a mirror of props :)

Reply:
Oh wow. I had no idea we were doing that :man_facepalming: